src/content/profile-tree.js

```diff
@@ -1,22 +1,18 @@
 // @flow
 import { timeCode } from '../common/time-code';
-import { getSampleFuncStacks } from './profile-data';
-import type { Thread, FuncTable, ResourceTable, StringTable, IndexIntoFuncTable } from '../common/types/profile';
+import { getSampleFuncStacks, resourceTypes } from './profile-data';
+import type { Node, Thread, FuncTable, ResourceTable, StringTable, IndexIntoFuncTable } from '../common/types/profile';
 import type { FuncStackTable, IndexIntoFuncStackTable, FuncStackInfo } from '../common/types/profile-derived';
 import type { Milliseconds } from '../common/types/units';
 
-type Node = {
-  totalTime: string,
-  totalTimePercent: string,
-  selfTime: string,
-  name: string,
-  lib: string,
-  dim: boolean,
-};
-
 type FuncStackChildren = IndexIntoFuncStackTable[];
 type FuncStackTimes = { selfTime: Milliseconds, totalTime: Milliseconds };
 
+function extractFaviconFromLibname(libname: string): string | null {
+  const url = new URL('/favicon.ico', libname);
```

Comment:
I did it in my previous patch, but now it's implicitely done in https://github.com/devtools-html/perf.html/pull/246/files/01798a369bf8120d12c0798252d6251861634f9a#diff-4d73a4ce02dae7e532bafe7333de9cd7R121

That said I'm not against it. I was a little bit afraid about the performance of doing it if we were reasonnably sure that we'd be safe, but maybe it's too implicit here.